scripts/travis/test.sh

```diff
@@ -2,6 +2,6 @@
 # pwd is the git repo.
 set -e
 
-export DJANGO_SETTINGS_MODULE=plugincheck.settings_travis
+export DJANGO_TEST_SETTINGS_MODULE=plugincheck.settings.travis
```

Comment:
Where does `plugincheck/settings/travis.py` get created? I don't see it anywhere, though I might be looking in all the wrong places.